pymilvus/orm/iterator.py

```diff
@@ -214,11 +231,17 @@ def __init__(
         self._filtered_ids = []
         self._filtered_distance = None
         self._schema = schema
+        self.__check_remove_limit()
         self.__check_metrics()
         self.__check_radius()
         self.__seek()
         self.__setup__pk_prop()
 
+    # as we use batch_size as the page size, so we remove LIMIT here
+    def __check_remove_limit(self):
+        if self._kwargs.get(LIMIT, 0) != 0:
+            self._kwargs.pop(LIMIT)
+
```

Comment:
Why delibretely remove "limit" key? would it be clearer just NOT USE IT?

Reply:
to avoid clients provide limit parameter when calling the method, otherwise the function will failed with exception 'multivalve for limit'

Reply:
see the comments below, key-value calls will avoid this exception, betther not use position calls on a funciton with so many parameters, could easy got wrong, and needed extra care for the kwargs.